storybook/src/components/Blockquote/Blockquote.stories.tsx

```diff
@@ -14,13 +14,19 @@ const meta = {
   component: Blockquote,
   args: {
     children: quote,
-    inverseColor: false,
   },
   argTypes: {
     children: {
       description: 'The text for the quote.',
       table: { disable: false },
     },
+    color: {
+      control: {
+        labels: { inverse: 'inverse', undefined: 'default' },
```

Comment:
Repeating the label for a property with the same name is unnecessary.

```
labels: { undefined: 'default' },
```

Reply:
This is to allow you disable the inverse radio  button.

Reply:
Indeed – my point is that the mapping `inverse: 'inverse'` can be left out.